include/deal.II/multigrid/mg_transfer.h

```diff
@@ -76,19 +76,18 @@ namespace internal
 
     static const bool requires_distributed_sparsity_pattern = false;
 
-    template <typename SparsityPatternType, typename DoFHandlerType>
+    template <typename SparsityPatternType, int dim, int spacedim>
     static void
     reinit(Matrix &matrix,
            Sparsity &,
            int                        level,
            const SparsityPatternType &sp,
-           DoFHandlerType &           dh)
+           DoFHandler<dim, spacedim> &dh)
```

Comment:
I think the tester complains because you should do
```suggestion
           const DoFHandler<dim, spacedim> &dh)
```